External/Plugins/FlashDebugger/Controls/BreakPointUI.cs

```diff
@@ -512,6 +557,63 @@ private void TstxtFilter_KeyDown(object sender, KeyEventArgs e)
             }
         }
 
+        #region Auto Completion
+
+        protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
+        {
+            // Ctrl+Space is detected at the form level instead of the editor level, so when we are docked we need to catch it before
+            if (keyData == (Keys.Control | Keys.Space) || keyData == (Keys.Control | Keys.Shift | Keys.Space))
```

Comment:
Because as a sample/obvious addition after making the decoupling, I added the completion list to some other places inside FD. That one is [this](http://imgbox.com/Oq6h9yN1).